docs/020-deployment/howtos/080-configure-dns.md

```diff
@@ -73,51 +73,49 @@ If you have any questions about DNS configuration, you can request CluedIn suppo
 
 **To update the DNS configuration for CluedIn**
 
-1. Download the current cluster configuration file by running the following command:
-```
-helm get values cluedin-platform -n cluedin -o yaml > Cluster-Current-values.yaml
-```
-2. Open the file in nano editor by running the following command:
-```
-nano Cluster-Current-values.yaml
-```
-3. In the file, find a line that looks similar to the one shown below. Look for the entry named **dns**. It is the configuration that controls the DNS address used by CluedIn.
-![configure-dns-1.png](../../assets/images/ama/howtos/configure-dns-1.png)
-4. Edit the value of the host name to reflect your main domain. If you have 3 environments (development, test, and production), update the value of the host name for each environment (`dev.mycompany.com`, `test.mycompany.com`, `prod.mycompany.com.`).
-5. Find the **TLS hosts** section. The example of the section is shown below.
-```yaml
-global:
-  ingress:
-    tls:
-      hosts:
-      - departmentX.20.0.189.11.sslip.io
-      - app.20.0.189.11.sslip.io
-      - clean.20.0.189.11.sslip.io
-      - '*.20.0.189.11.sslip.io'
-```
-6. Replace the hosts section as shown below.
-```yaml
-global:
-  ingress:
-    tls:
-      hosts:
-      - departmentX.mycompany.com
-      - app.mycompany.com
-      - clean.mycompany.com
-      - '*.mycompany.com'
-```
-
-7. If you are using multiple environments (development, test, and production), update the TLS hosts section for each environment in the `values.yaml` file:
+1. Download the current cluster configuration file by running the following command:  
+    `helm get values cluedin-platform -n cluedin -o yaml > Cluster-Current-values.yaml`
+1. Open the file in a text editor by running the following command
```

Comment:
Looks like the command is missing here

Reply:
Good spot, I removed it to be fair as it was assuming the reader was running a specific version. Tried to generalise it :)